tensorflow/core/ops/mkl_nn_ops.cc

```diff
@@ -473,9 +470,8 @@ REGISTER_OP("_MklNativeMaxPool3D")
     .Attr("workspace_enabled: bool = false")
     .SetShapeFn(shape_inference::Pool3DShape)
     .Doc(R"doc(
-MKL version of MaxPool3D operator that does not depend
-on layout propagation. Uses oneDNN APIs to perform 3D max pooling
-on the input.
+oneDNN version of MaxPoolGrad that does not depend on layout propagation.
+Uses oneDNN APIs to compute gradients of MaxPool operator.
```

Comment:
Nit: Message is for the wrong op.
```suggestion
oneDNN version of MaxPool3D operator that does not depend on layout propagation.
Uses oneDNN APIs to perform 3D max pooling on the input.
```

Reply:
Sorry for that. Fixed.